bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "echo-angular",
-  "version": "1.1.0",
+  "version": "2.0.0",
```

Comment:
If someone gets this version they will get a breaking change on app-logger-angular as it stands.  Safer to make this clear on this I reckon.